data/sql/V1__create.sql

```diff
@@ -1,35 +1,103 @@
--- MySQL Script generated by MySQL Workbench
```

Comment:
The idea of flyway is to migrate changes, so they are versionable. Since this is the first version, and we haven't deployed it anywhere, it's ok to have it like this, but in the future, create a V2__.... etc.